kevlar/novel.py

```diff
@@ -21,29 +21,38 @@ class KevlarCaseSampleMismatchError(ValueError):
 
 
 def kmer_is_interesting(kmer, casecounts, controlcounts, case_min=5,
-                        ctrl_max=1):
+                        ctrl_max=1, screen_thresh=None):
     """
     Well, is it?
 
     Consult the k-mer's abundance in each sample to determine whether it is
     "interesting". It must be >= `case_min` in each of `casecounts`, and must
     be <= `ctrl_max` in each of `controlcounts`.
+
+    Returns 4 values: 2 booleans, and 2 lists of integers
+    - boolean indicating whether the k-mer is interesting
+    - boolean indicating whether the entire read should be discarded
+    - list of case sample abundances (empty if k-mer not interesting)
+    - list of control sample abundances (empty if k-mer not interesting)
```

Comment:
Cleaned up this entire function to make its return signature consistent despite the result.